api/src/main/resources/messages.properties

```diff
@@ -1,102 +1,92 @@
+patientflags.editFlag.customEvaluator=- FlagEvaluator class:
+patientflags.manageTags.addTag=Add a Tag
+patientflags.manageTags.deleteTag=Delete Tag
+patientflags.manageTags.deleteTagConfirm=You are about to completely remove this Tag from the system.
+patientflags.editTag.saveTag=Save Tag
+patientflags.editTag.visibleTo=Visible To
+patientflags.editTag.shownIn=Shown In
+patientflags.managePriorities.addPriority=Add Priority
+patientflags.managePriorities.deletePriority=Delete Priority
+patientflags.managePriorities.deletePriorityConfirm=You are about to completely remove this Priority from the system.
+patientflags.editPriority.savePriority=Save Priority
+patientflags.findFlaggedPatients.byTagSet=Find Flagged Patients by Tag Sets
+patientflags.findFlaggedPatients.selectFlag=Select a Flag
+patientflags.findFlaggedPatients.selectTag=Select Tag(s)
+patientflags.findFlaggedPatients.anyTag=flags that contain any selected tags
+patientflags.findFlaggedPatients.allTags=only flags that contain all selected tags
+patientflags.managePatientFlagsProperties.patientHeaderDisplay=Display flags in the header of Patient Dashboard (requires a release of OpenMRS after 1.5.1)
+patientflags.managePatientFlagsProperties.patientOverviewDisplay=Display flags in overview section of Patient Dashboard
+patientflags.managePatientFlagsProperties.username=Allow Groovy flags all privileges associated with this user
+patientflags.managePatientFlagsProperties.success=Flag global properties successfully updated. If you have toggled either of the display areas on or off, please restart the module for the changes to take effect.
+patientflags.flagpatientdashboard.header=Flags:
+patientflags.errors.noName=Name is required
+patientflags.errors.noCriteria=Criteria is required
+patientflags.errors.invalidCriteria=Criteria is invalid for selected flag type
+patientflags.errors.noPatientIdCriteria=Criteria must contain '*.patient_id'
+patientflags.errors.noTag=Tag name is required
+patientflags.errors.noEvaluator=Please select an Evaluator
+patientflags.errors.nameTooLong=Name can only be 255 characters in length
+patientflags.errors.criteriaTooLong=Criteria can only be 5000 characters in length
+patientflags.errors.messageTooLong=Message can only be 255 characters in length
+patientflags.errors.tagTooLong=Tag can only be 255 characters in length
+patientflags.errors.flagNameExists=A flag with this name already exists
+patientflags.errors.tagNameExists=A tag with this name already exists
+patientflags.errors.noPriority=Priority is required
+patientflags.errors.noPriorityName=Priority name is required
+patientflags.errors.noStyleName=Priority style is required
+patientflags.errors.noRank=Rank is required
+patientflags.errors.priorityNameTooLong=Name can only be 255 characters in length
+patientflags.errors.noUniqueName=Priority name already exists. Change name to unique
```

Comment:
We can improve the alert message in particular for the last part of it(`Change name to unique`). It seems to suggest the name should be changed to **unique**

Reply:
So what exactly message should be? Name should be changed to unique that's why I added such message

Reply:
> Priority name already exists. Change name to unique

The point here is that the user is likely going to type the name `unique` as a priority name based on the alert message.

How about if we use `Priority name already exists. Change to another name`?

Reply:
ah ok I see your point that user can try to type 'unique' as a name. ok I will change message into 'Priority name already exists. Change to another name'

Reply:
I've already fixed it